gradle.properties

```diff
@@ -18,4 +18,5 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+android.jetifier.ignorelist=android-base-common,common
```

Comment:
Can we get rid of the jetifier altogether? (outside the scope of this I guess)

Reply:
That's a good question, I had to add this to suppress an error? I don't even remember. Will experiment.

Reply:
must have done the replacements. it was no longer needed